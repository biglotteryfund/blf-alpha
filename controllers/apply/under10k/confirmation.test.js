/* eslint-env jest */
'use strict';
const confirmationBuilder = require('./confirmation');

test('should return confirmation text based on country', () => {
    const england = confirmationBuilder({
        locale: 'en',
        data: {
            projectCountry: 'england',
            mainContactName: { firstName: 'Bob', lastName: 'Example' },
            mainContactEmail: 'bob@example.com',
            seniorContactName: { firstName: 'Ann', lastName: 'Example' },
            seniorContactEmail: 'ann@example.com',
        },
    });

    expect(england.body).toMatchSnapshot();

    const scotland = confirmationBuilder({
        locale: 'en',
        data: {
            projectCountry: 'scotland',
            mainContactName: { firstName: 'Bob', lastName: 'Example' },
            mainContactEmail: 'bob@example.com',
            seniorContactName: { firstName: 'Ann', lastName: 'Example' },
            seniorContactEmail: 'ann@example.com',
        },
    });

    expect(scotland.body).toMatchSnapshot();
});
