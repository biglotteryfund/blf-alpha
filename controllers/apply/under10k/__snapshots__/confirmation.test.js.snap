// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return confirmation text based on country 1`] = `
<h2>
  We’ve just sent an email to your main and senior contact
</h2>
<p>
  This is just a confirmation email, with a summary of your answers
(in case you want to look back at them at any point).
</p>
<h2>
  Now we’ve got your application – 
we'll start assessing it as soon as we can
</h2>
<p>
  We’ll look at your idea and do some checks.
Given the huge demand for funding,
we’re focusing on funding projects and organisations
helping communities through the COVID-19 pandemic,
so they can start as soon as possible
</p>
<h2>
  While we’re assessing your application – we might get in touch
</h2>
<p>
  We don’t always do this. It’s only if we need a bit more information.
So don’t worry if you don’t hear from us.
</p>
<h2>
  We’ll let
  <span data-hj-suppress>
    Bob Example and
Ann Example
  </span>
  know our decision by email
</h2>
<p>
  We’ll email
  <strong data-hj-suppress>
    bob@example.com
  </strong>
  and
  <strong data-hj-suppress>
    ann@example.com
  </strong>
  .
Make sure to check the junk mail too (just in case).
</p>
<h2>
  If you have any questions, you can contact us in the meantime
</h2>
<p>
  Please call 028 9568 0143 or email
  <a href="mailto:afe@tnlcommunityfund.org.uk">
    afe@tnlcommunityfund.org.uk
  </a>
  and we will be happy to help you.
</p>
<p>
  Good luck with your application!
  <br>
  <strong>
    The National Lottery Community Fund
  </strong>
</p>
`;

exports[`should return confirmation text based on country 2`] = `
<h2>
  We’ve just sent an email to your main and senior contact
</h2>
<p>
  This is just a confirmation email, with a summary of your answers
(in case you want to look back at them at any point).
</p>
<h2>
  Now we’ve got your application – 
we'll start assessing it as soon as we can
</h2>
<p>
  We’ll look at your idea and do some checks.
We’re now prioritising decisions for COVID-19 related
projects, so they can start sooner. And it might take us longer
to assess applications that aren’t about COVID-19.
</p>
<h2>
  While we’re assessing your application – we might get in touch
</h2>
<p>
  We don’t always do this. It’s only if we need a bit more information.
So don’t worry if you don’t hear from us.
</p>
<h2>
  We’ll let
  <span data-hj-suppress>
    Bob Example and
Ann Example
  </span>
  know our decision by email
</h2>
<p>
  We’ll email
  <strong data-hj-suppress>
    bob@example.com
  </strong>
  and
  <strong data-hj-suppress>
    ann@example.com
  </strong>
  .
Make sure to check the junk mail too (just in case).
</p>
<h2>
  If you have any questions, you can contact us in the meantime
</h2>
<p>
  Please call 0141 846 0447 or email
  <a href="mailto:advicescotland@tnlcommunityfund.org.uk">
    advicescotland@tnlcommunityfund.org.uk
  </a>
  and we will be happy to help you.
</p>
<p>
  Good luck with your application!
  <br>
  <strong>
    The National Lottery Community Fund
  </strong>
</p>
`;
